src/public/js/github/github.js

```diff
@@ -0,0 +1,13 @@
+import orgs from './orgs.js';
+import repos from './repos.js';
+import issues from './issues.js';
+
+export default (view, payload) => {
+  if (view === 'orgs') {
+    orgs(payload);
```

Comment:
They are in the same view, I've called the view `github`. I've just further modularised each part into `orgs`, `repos`, `issues` and `timer`.

Yes it would be good to have a look at the wireframe. Although, at the moment, the front end is such that it is really flexible to change how it will look.